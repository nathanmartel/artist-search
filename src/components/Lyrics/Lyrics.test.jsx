import React from 'react';
import { shallow } from 'enzyme';
import Lyrics from './Lyrics';

describe('Lyrics component', () => {
  it('renders Lyrics', () => {
    const wrapper = shallow(<Lyrics id={'idstring'} title={'Fitter Happier'} length={'12345'} />);
    expect(wrapper).toMatchSnapshot();
  });
});
