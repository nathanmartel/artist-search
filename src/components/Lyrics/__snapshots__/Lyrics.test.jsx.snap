// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Lyrics component renders Lyrics 1`] = `
<li
  key="idstring"
>
  <Link
    to="/song/idstring"
  >
    Fitter Happier
     (
    12345
    )
  </Link>
</li>
`;
